filecoin/test/filecoin-service.test.js

```diff
@@ -115,7 +128,9 @@ for (const [title, unit] of Object.entries(filecoinApiTest.service.storefront))
       dealTrackerSigner,
       service
     ).connection
-
+    const pdpStore = createPDPStore()
+    const storageProviders = [await StorageNode.activate({ pdpStore })]
+    const router = createRouter(storefrontSigner, storageProviders)
```

Comment:
Ah, so to check my understanding - we need these in the tests because the service will now make an invocation to the node that issued the `pdp/accept` invocation to get the `pdp/info` for the piece.

We just need to ensure the delegation that the node provides to the upload service includes `pdp/info`...

Reply:
yes